### PR TITLE
teams/migration: a vote for maxshuang as the committer

### PR DESCRIPTION
**Pull request by amyangfei**
As title said

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL, thanks!
/cc @lance6716 @overvenus @liuzix

File: votes/0670-maxshuang-as-migration-comitter.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# A Vote for maxshuang as Migration committer
+
+## Proposal
+
+[@maxshuang](https://github.com/maxshuang) has been promoted to the reviewer of tiflow half a year ago.
+
+In the last six months, @maxshuang is leading the projects of `dataflow engine metastore` and `dataflow engine multi-tenant integration with TiDB cloud`, both of these two projects are the core parts of dataflow engine. He also takes part in the DDL related work in TiCDC, finds several DDL related bugs and fixes them.
+
+The Works show the contribution of @maxshuang as well as his ability and responsibility to `pingcap/tiflow` project. The details of all his PRs and reviews are:
```
Comment (lance6716) [inline votes/0670-maxshuang-as-migration-comitter.md:9]:
```suggestion
The works show the contribution of @maxshuang as well as his ability and responsibility to `pingcap/tiflow` project. The details of all his PRs and reviews are:
```

**Review by lance6716 — APPROVED** [review]
PR number in vote file should be 694

**Review by overvenus — APPROVED** [review]
Congrats!

**Pull request merged**